packages/devtools_app/integration_test/test/live_connection/eval_and_browse_test.dart

```diff
@@ -199,13 +200,15 @@ class _EvalAndBrowseTester {
   ///
   /// If [next] is provided, will repeat the tap untill [next] returns results.
   /// If [next] is not null returns [next].
-  Future<Widget?> tapAndPump(
+  Future<Finder?> tapAndPump(
```

Comment:
can you update the dart doc for this helper? "If [next] is provided" and "If [next] is not null" describe the same case.